src/bot/services.py

```diff
@@ -42,30 +42,32 @@ async def notify_declined_request(
             )
         await self.__bot.send_message(user.telegram_id, text)
 
-    async def notify_approved_task(self, user: models.User, report: models.Report) -> None:
+    async def notify_approved_task(self, user: models.User, report: models.Report, shift: models.Shift) -> None:
         """Уведомление участника о проверенном задании.
 
         - Задание принято, начислен 1 ломбарьерчик.
         """
-        photo_date = dt.strftime(report.uploaded_at, FORMAT_PHOTO_DATE)
+        photo_date = datetime.strftime(report.uploaded_at, FORMAT_PHOTO_DATE)
         text = (
             f"Твой отчет от {photo_date} принят! "
             f"Тебе начислен 1 \"ломбарьерчик\". "
             f"Следующее задание придет в 8.00 мск."
         )
+        if date.today() >= shift.finished_at:
+            text = text.replace("Следующее задание придет в 8.00 мск.", "")
         await self.__bot.send_message(user.telegram_id, text)
 
-    async def notify_declined_task(self, user: models.User) -> None:
+    async def notify_declined_task(self, user: models.User, shift: models.Shift) -> None:
         """Уведомление участника о проверенном задании.
 
         - Задание не принято.
         """
         text = (
             "К сожалению, мы не можем принять твой фотоотчет! "
             "Возможно на фотографии не видно, что именно ты выполняешь задание. "
-            "Предлагаем продолжить, ведь впереди много интересных заданий. "
-            "Следующее задание придет в 8.00 мск."
         )
+        next_task = "Предлагаем продолжить, ведь впереди много интересных заданий. Следующее задание придет в 8.00 мск."
+        text = (text + next_task) if date.today() < shift.finished_at else text

```

Comment:
Я бы так сделал. Да. Все рекомендуют использовать join, но он только запутает код и сделает более сложным.
```suggestion
        if date.today() < shift.finished_at:
            text = text + "Предлагаем продолжить, ведь впереди много интересных заданий. Следующее задание придет в 8.00 мск."

```